influxalchemy/meta.py

```diff
@@ -1,7 +1,33 @@
 """ InfluxDB Meta Measurement. """
 
+import sys
+from datetime import date
+
 from . import operations
 
+if sys.version_info.major >= 3:
+    # pylint: disable=no-name-in-module
+    from datetime import timezone
+    # pylint: disable=invalid-name
+    utc = timezone.utc
+else:
+    import pytz
+    # pylint: disable=invalid-name,redefined-variable-type
+    utc = pytz.utc
+
+
+def make_tz_aware(datetime_obj):
+    """ Make a date/datetime object to be timezone-aware """
+    # 'date' object doesn't need to be timezone aware
+    # pylint: disable=unidiomatic-typecheck
+    if type(datetime_obj) is date:
```

Comment:
It can be demonstrated as following:

```
>>> from datetime import date, datetime

>>> date_only = date.today()

>>> date_with_time = datetime.now()

>>> type(date_only) is date
True

>>> type(date_with_time) is date
False

>>> isinstance(date_only, date)
True

>>> isinstance(date_with_time, date)
True
```
So, `isinstance` does not strictly distinguish `date` and `datetime`, whereas `type(...) is` does.